migrator/migrations/m_195_to_m_196_vuln_request_users/migration_impl.go

```diff
@@ -40,6 +40,13 @@ func updateGlobalScope(ctx context.Context, database *types.Databases) error {
 		if approvers := obj.GetApprovers(); len(approvers) > 0 {
 			updated.ApproversV2 = convertSlimUserToApprovers(obj.GetApprovers())
 		}
+		if obj.GetDeferralReq() != nil && obj.GetDeferralReq().GetExpiry() != nil {
+			if obj.GetDeferralReq().GetExpiry().GetExpiresWhenFixed() {
+				updated.GetDeferralReq().Expiry.ExpiryType = storage.RequestExpiry_ANY_CVE_FIXABLE
```

Comment:
If the v1 deferral requests had just one CVE, the migrated v2 deferral request would also have one CVE. So why do we set the ExpiryType to `ANY_CVE_FIXABLE` and not `ALL_CVE_FIXABLE` here? While the end result would continue to remain the same, I just want to know the reasoning behind choosing one over the other.

Reply:
This is because the v1 unsnooze implementation behaved for `ANY_CVE_FIXABLE` expiry.  We have to ensure backward compatibility. For example, if after the upgrade, the request is updated with more CVEs and the central is rolled back, then the exception vs actuality would be different.